[2024-04-20T15:47:46.231+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T15:47:46.289+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-20T15:47:46.311+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-20T15:47:46.317+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T15:47:46.560+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): second_task> on 2024-04-14 00:00:00+00:00
[2024-04-20T15:47:46.609+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1872) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T15:47:46.604+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpd3dwlenn']
[2024-04-20T15:47:46.624+0000] {standard_task_runner.py:63} INFO - Started process 1883 to run task
[2024-04-20T15:47:46.626+0000] {standard_task_runner.py:91} INFO - Job 280: Subtask second_task
[2024-04-20T15:47:46.732+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2024-04-14T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T15:47:46.822+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-20T15:47:46.825+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T15:47:46.857+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T15:47:46.860+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after']
[2024-04-20T15:47:46.871+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T15:47:46.875+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after
[2024-04-20T15:47:46.875+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T15:47:46.876+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T15:47:46.897+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20240414T000000, start_date=20240420T154746, end_date=20240420T154746
[2024-04-20T15:47:46.921+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T15:47:46.935+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T15:47:46.936+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
